napari/layers/labels/labels.py

```diff
@@ -113,6 +113,8 @@ class Labels(_ImageBase):
         'translucent' renders without lighting. 'iso_categorical' uses isosurface
         rendering to calculate lighting effects on labeled surfaces.
         The default value is 'iso_categorical'.
+    depiction : str
```

Comment:
Can you please list the supported modes here?

Reply:
(Probably the same comment for the Image layer docstring)

Reply:
:+1: I would get pretty murdery if I came across the docstring below in an unfamiliar library :joy: